scopes/workspace/workspace/bit-map.ts

```diff
@@ -192,19 +192,24 @@ export class BitMap {
       const config = componentMap.config;
       if (!config) return;
       Object.keys(config).forEach((aspectId) => {
-        if (aspectId === sourceId.toString()) {
+        const fullSourceId = sourceId.toStringWithoutVersion();
```

Comment:
how `sourceId.toStringWithoutVersion();` becomes full source id?

Reply:
because it is passed as componentId as you can see in the method params:
```
 async rename(sourceIdStr: string, targetName: string, options: RenameOptions): Promise<RenameDependencyNameResult> {
    if (!isValidIdChunk(targetName)) {
      if (targetName.includes('.'))
        throw new Error(`error: new-name argument "${targetName}" includes a dot.
make sure this argument is the name only, without the scope-name. to change the scope-name, use --scope flag`);
      throw new InvalidName(targetName);
    }
    const sourceId = await this.workspace.resolveComponentId(sourceIdStr);   <-----
```

Reply:
ok, I understand, in that case the name of your var is confusing, it's not a full source id, but soruceIdWithoutVersion